test/concurrency/transaction_test.cpp

```diff
@@ -78,7 +68,7 @@ void CheckTxnLockSize(Transaction *txn, size_t shared_size, size_t exclusive_siz
 }
 
 // NOLINTNEXTLINE
-TEST_F(TransactionTest, DISABLED_SimpleInsertRollbackTest) {
+TEST_F(TransactionTest, SimpleInsertRollbackTest) {
```

Comment:
need to disable 🤪 this file differs from the private repo in `DISABLED_` prefix, and thus I added the private header.